dashboard/src/components/DeploymentForm/DeploymentFormBody/BasicDeploymentForm/TabularSchemaEditorTable/TabularSchemaEditorTableRenderer.tsx

```diff
@@ -102,15 +110,17 @@ export function renderConfigKey(value: IBasicFormParam, row: any, _saveAllChange
               <></>
             )}
           </CdsButton>
-          {renderCellWithTooltip(value, "key", "breakable self-center", true, MAX_LENGTH / 1.5)}
+          {renderCellWithTooltipBase(stringKey, "breakable self-center", true, MAX_LENGTH / 1.5)}
         </div>
       </>
     </div>
   );
 }
 
 export function renderConfigType(value: IBasicFormParam) {
-  return renderCellWithTooltip(value, "type", "italics");
+  const stringType =
+    value?.type === "array" ? `${value?.type}<${value?.items?.type}>` : value?.type;
```

Comment:
```suggestion
    value?.type === "array" ? `array of ${value?.items?.type}` : value?.type;
```
Minor suggestion, as it is for human reading.

Reply:
Ok, done (in a follow-up PR)

After:
![image](https://user-images.githubusercontent.com/11535726/197246149-6cd8904a-bf35-4d13-977e-10ea1a5b57b7.png)

Before:
![image](https://user-images.githubusercontent.com/11535726/197246229-f92b4a58-4232-45a2-92b3-38b1f4429b4e.png)